### ISSUE TITLE
complete model prototyping and fine tuning 

### ISSUE THREAD

**Issue by poipiii** [poipiii/pytorch-cnn-experiment#4]
- [x] finish building  model training and evaluation loop
- [x] **fix issue with random split giving unreproducible training results** 
- [x] add wandb and tensorboardx logging 
- [ ] **fix issue with weird validation accuracy and loss values**
- [ ] test and tune model parameters
